detection-rules/impersonation_microsoft.yml

```diff
@@ -14,17 +14,17 @@ source: |
       ilike(subject.subject, '*Microsoft 365*') 
       and ilike(subject.subject, '*is expired*')
     )
-    or iregex_search(body.html.raw, ".*reply in teams.*", ".*reach.*you.*microsoft.*teams.*", ".*install microsoft teams.*")
+    or iregex_search(body.html.raw, ".*reach.*you.*microsoft.*teams.*")
     or ilike(sender.display_name, '*new activity in Teams*')
     or subject.subject =~ 'Offline Message in Teams'
     or ilike(subject.subject, '*Teams Sent A Message')
     or sender.display_name in~ ('Microsoft Partner Network','Microsoft',
                                'Microsoft Feedback', 'Microsoft account team',
-                               'Microsoft Support', 'Microsoft Outlook', 
-                               'Microsoft 365 Message center', 
+                               'Microsoft Support', 'Microsoft 365 Message center', 
                                'Microsoft Azure')
-  )                                                                                                                                                                          
+  )                                                                                                                         
   and sender.email.domain.root_domain != 'microsoft.com'
```

Comment:
[According to Microsoft](https://docs.microsoft.com/en-us/troubleshoot/azure/general/email-domains-support-agent), emails can also come from `microsoftsupport.com`